contracts/staking/contracts/src/interfaces/IStructs.sol

```diff
@@ -21,24 +21,6 @@ pragma solidity ^0.5.9;
 
 interface IStructs {
 
-    /// @dev Allowed signature types.
-    enum SignatureType {
```

Comment:
this must've slipped past in [2104](https://github.com/0xProject/0x-monorepo/pull/2104) 😨